src/iface-modbus.c

```diff
@@ -224,10 +277,17 @@ int iface_modbus_read_data(int reg_addr, int bit_offset, knot_value_type *out)
 	case TYPE_U32:
 		rc = modbus_read_registers(modbus_ctx, reg_addr, 2,
 					   (uint16_t *) &tmp.val_u32);
+		iface_modbus_config_endianness_type_recv_32_bits(&tmp.val_u32,
+						endianness_type);
+
 		break;
+
```

Comment:
Please, remove this blank line on commit 87a5a0e62f251836807d5cf897fa3d759201f91a.